cypress/integration/task-3.js

```diff
@@ -0,0 +1,46 @@
+import TimeLoggingPage from '../pom/timeLoggingPageObject'
+import NavBar from '../pom/navBarObject'
+
+const timeLoggingPage = new TimeLoggingPage()
+const navBar = new NavBar()
+
+describe('Task 2', function() {
+
+    it('T-2: Users should be able to log in with "Team Lead" role and see appropriate tabs', function () {
+
+        let roles = ["User", "Team Lead", "Manager", "Accountant", "Admin"]
+
+        const nav_length = {
+            "User" : 1,
+            "Team Lead" : 2,
+            "Manager" : 5,
+            "Accountant" : 5,
+            "Admin" : 6,
+        }
+        const user = "TestCon User 6"
+        let i = 0
+
+        while (roles[i]){
```

Comment:
I have my doubts about this while loop approach, there might be a cleaner way to do this (array mapping, foreach, etc). But is not a blocker for this pull requests

Reply:
Definitely could be improved, didn't have much time to refactor it properly and my javascript skills can also be improved! :) Thanks for you feedback.